kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -52,6 +52,8 @@
     <version.tomcat8>8.5.12</version.tomcat8>
     <version.wildfly10>10.1.0.Final</version.wildfly10>
     <version.wildfly11>11.0.0.Final</version.wildfly11>
+    <!-- Need to override jboss-remoting version to be compatible with jboss-remote-naming -->
+    <version.org.jboss.remoting>4.0.18.Final</version.org.jboss.remoting>
```

Comment:
@mswiderski Since you have lots of recent commits in this directory, would you mind reviewing this? (Or recommend someone else if relevant?)

The ip-bom upgrade in the related PRs updates wildfly-core, which required updating jboss-remoting to version 5.0.3.Final, but unfortunately the most recent jboss-remote-naming release is only compatible with jboss-remoting 4.0.18.Final.

As far as I can tell, jboss-remote-naming is only used in tests as a transitive dependency of wildfly-jms-client-bom, so I think this change. Am I missing anything?

//cc @psiroky @mariofusco @manstis @mbiarnes

Reply:
@mbarkley as long as it works then it's fine 

@sutaakar could you please have a look when you have chance?

Reply:
Okay. This change got us from many test failures down to just 2. Looking into those now.

Reply:
Looks good to me. Those 2 failures seems to be caused by behaviour change caused by dependency updates. Seems that https://github.com/kiegroup/droolsjbpm-integration/blob/494cf10855b338ff2472c17a4cc3e9e902c7920d/kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/proxy/KieServerProxyClient.java#L148 needs to be updated to catch this new exception.

Reply:
@sutaakar thanks for pointing that out. I will update the PR shortly.